CHANGELOG.md

```diff
@@ -5,12 +5,21 @@
 accidentally triggering the load of a previous DB version.**
 
 ## To be released
+Users who upgrade from 1.7 and have continuous aggregateswith real time support set up should follow these instructions
+
+Step 1: Find views that have real time support enabled
+select view_name, materialized_only from timescaledb_information.continuous_aggregates where materialized_only = 'true';
+
+Step 2: Re-enable real time support for the views from
+Step 1.
+ALTER VIEW <view_name> SET (timescaledb.materialized_only=false);
```

Comment:
Why dont we do this in the update script instead of requiring the user to fix it?

```
CREATE OR REPLACE FUNCTION realtime_agg_update() returns void LANGUAGE PLPGSQL AS
$f$
DECLARE
  r RECORD;
BEGIN
    FOR r IN select view_name from timescaledb_information.continuous_aggregates where materialized_only = false
    LOOP 
		EXECUTE FORMAT('ALTER VIEW %I SET (timescaledb.materialized_only=false)', r.view_name);
    END LOOP;
END;
$f$;

SELECT realtime_agg_update();
DROP FUNCTION realtime_agg_update();
```

Reply:
@svenklemm Yes, agree , that this should be automated and not a manual step. Could you point me to the upgrade script where this should be added for automation?